cmake/Install.cmake

```diff
@@ -361,7 +361,6 @@ install(
 
 # TODO Remove version from file name or generate
 # These files names are stored in variables because they also appear as start menu shortcuts later.
-set(RULES_XLS Rules22-2-0-to-22-3-0.md)
```

Comment:
This isn't used, and was wrong anyway.  If we want to use variables, we should just use the raw E+ version variables.